widgetssdk/src/main/java/com/glia/widgets/chat/adapter/holder/CustomCardViewHolder.java

```diff
@@ -0,0 +1,69 @@
+package com.glia.widgets.chat.adapter.holder;
+
+import android.view.View;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.glia.androidsdk.chat.ChatMessage;
+
+/**
+ * CustomCardViewHolder describes an item view
+ * <p>
+ * The CustomCardViewHolder implementation should have fields to cache
+ * the potentially costly View.findViewById(int) results.
+ * <p>
+ * <b>Usage example:</b>
+ * <pre>{@code
+ * class ExampleViewHolder extends CustomCardViewHolder {
+ *     private final TextView textView;
+ *     private final Button button;
+ *
+ *     public ExampleViewHolder(@NonNull View itemView, @NonNull UiTheme uiTheme) {
+ *         super(itemView);
+ *         this.textView = itemView.findViewById(R.id.text_view);
+ *         this.button = itemView.findViewById(R.id.button);
+ *     }
+ *
+ *     @Override
+ *     public void bind(@NonNull ChatMessage message, @NonNull ResponseCallback callback) {
+ *         messageTextView.setText(message.getContent());
+ *         try {
+ *             String customField = metadata.getString("customField");
```

Comment:
No metadata object in the scope
```suggestion
 *             String customField = message.getMetadata().getString("customField");
```

Reply:
fixed